website/content/en/preview/upgrading/v1beta1-reference.md

```diff
@@ -604,23 +604,49 @@ In addition to these metrics, the MachineNotFound error returned by the `karpent
-| enableENILimitedPodDensity                        | Dropped                         | Dropped                       |
-| isolatedVPC                                       | ISOLATED_VPC                    | --isolated-vpc                |
-| vmMemoryOverheadPercent                           | VM_MEMORY_OVERHEAD_PERCENT      | --vm-memory-overhead-percent  |
-| interruptionQueueName                             | INTERRUPTION_QUEUE_NAME         | --interruption-queue-name     |
-| reservedENIs                                      | RESERVED_ENIS                   | --reserved-enis               |
-| featureGates.enableDrift                          | FEATURE_GATE="Drift=true"       | --feature-gates Drift=true    |
+| **`karpenter-global-settings` ConfigMap Key** | **Environment Variable**   | **CLI Argument**
+|-----------------------------------------------|----------------------------|--------------------------|
+| batchMaxDuration                              | BATCH_MAX_DURATION         | --batch-max-duration     |
+| batchIdleDuration                             | BATCH_IDLE_DURATION        | --batch-idle-duration    |
+| assumeRoleARN                                 | ASSUME_ROLE_ARN            | --assume-role-arn        |
+| assumeRoleDuration                            | ASSUME_ROLE_DURATION       | --assume-role-duration   |
+| clusterCABundle                               | CLUSTER_CA_BUNDLE          | --cluster-ca-bundle      |
+| clusterName                                   | CLUSTER_NAME               | --cluster-name           |
+| clusterEndpoint                               | CLUSTER_ENDPOINT           | --cluster-endpoint       |
+| defaultInstanceProfile                        | Dropped                    | Dropped                  |
+| enablePodENI                                  | Dropped                    | Dropped                  |
+| enableENILimitedPodDensity                    | Dropped                    | Dropped                  |
+| isolatedVPC                                   | ISOLATED_VPC               | --isolated-vpc           |
+| vmMemoryOverheadPercent                       | VM_MEMORY_OVERHEAD_PERCENT | --vm-memory-overhead-percent |
+| interruptionQueueName                         | INTERRUPTION_QUEUE         | --interruption-queue     |
+| reservedENIs                                  | RESERVED_ENIS              | --reserved-enis          |
+| featureGates.driftEnabled                     | FEATURE_GATE="Drift=true"  | --feature-gates Drift=true |
+
+## Helm Values
+
+The v1beta1 helm chart comes with a number of changes to the values that were previously used in v0.31.x. Your older helm values will continue to work throughout v0.32.x but any values no longer specified in the chart will no longer be supported starting in v0.33.0.
+
+| < v0.32.x Key                           | >= v0.32.x Key                   |
```

Comment:
thoughts on this?
```suggestion
|  Before v0.32.x Key                           | v0.32.x+ Key                   |
```

Reply:
I personally like the way that it is right now.